### PR TITLE
chore: update tech-radar to 1.0

### PR DESCRIPTION
**Pull request by 04kash**
## Description

**Breaking Changes**

- The Tech Radar plugin now requires the `@backstage-community/tech-radar-backend` package for loading data.
- To use the Tech Radar plugin, you must enable both `@backstage-community/tech-radar-backend` and `@backstage-community/tech-radar`.

### Additional Configuration

To enable proper data loading, follow these configuration steps:

1. **Specify the Data URL**  
   Add the following to your `app-config.yaml` file to provide the URL for fetching JSON data:
   
   ```yaml
   techRadar:
     url: https://github.com/backstage/community-plugins/blob/main/workspaces/tech-radar/plugins/tech-radar-common/src/sampleTechRadarResponse.json

2. **Configure Integrations**
In the app-config.yaml, configure integrations for the Version Control System (VCS) from which the data will be fetched, under the integrations section. See: https://backstage.io/docs/integrations/ 


## Which issue(s) does this PR fix

- Fixes: https://issues.redhat.com/browse/RHIDP-4759 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

## How to test changes / Special notes to the reviewer

### CODE REVIEW

**Comment by 04kash** [comment]
Tested it and it works as expected.

File: dynamic-plugins/wrappers/backstage-community-plugin-tech-radar-backend-dynamic/src/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export {default} from '@backstage-community/plugin-tech-radar-backend';
```
Comment (schultzp2020) [inline dynamic-plugins/wrappers/backstage-community-plugin-tech-radar-backend-dynamic/src/index.ts:1]:
```suggestion
export {default} from '@backstage-community/plugin-tech-radar-backend';

```

File: github-app-backstage-showcase-04kash-app-credentials.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Name: Backstage-Showcase-04kash-App
+appId: 1042813
+webhookUrl: https://smee.io/nMKkctYLlh0UT40PZI
+clientId: Iv23lip9Rb05btAEEy6C
+clientSecret: d10125c1fb04afac3d0f0e874ab467eadcdfefe9
+webhookSecret: 3b8b39ab9c39fbb6306c461e5871b9da483d9585
+privateKey: |
+  -----BEGIN RSA PRIVATE KEY-----
```
[the remark this answers is not in the document]
Comment (schultzp2020) [reply github-app-backstage-showcase-04kash-app-credentials.yaml]: @04kash don't forget to delete these creds regen new ones.

**Review by schultzp2020 — APPROVED** [review]
/lgtm

**Pull request merged**